### DISCUSSION TITLE
Remembering window size, position, and screen

### DISCUSSION THREAD

**Discussion by HEIC-to-JPEG-Dev** [dotMorten/WinUIEx#189] [General]
Is this capability built into this library? Or do I have to write that myself?

**Comment by dotMorten** [comment]
It is built in for packaged apps. For unpackaged apps you'll need to provide a way to store the setting somewhere.
Look for Window Manager.PersistenceId and set it and you're good to go

**Comment by HEIC-to-JPEG-Dev** [comment]
Thank you, I’ll look there

**Comment by HEIC-to-JPEG-Dev** [comment]
Looks like that's the solution for me, thank you.

Can I also say that yours is the first github open source project I've downloaded and loaded into VS that actually works - Even "Files" doesn't and many others without a special config at the developers end. Great Work, If my new app starts to sell, I'll be sponsoring.

**Reply by dotMorten** [reply]
Appreciate the kind words.
